tasks/Darwin.yml

```diff
@@ -53,8 +53,16 @@
     state: absent
 
 - name: Install homebrew
-  ansible.builtin.include_role:
-    name: geerlingguy.mac.homebrew
+  block:
+    - include_role:
```

Comment:
should this be `ansible.builtin.include_role`?

Reply:
updated, thanks